### PR TITLE
fix(messaging): Sync with status-go content type enum

### PR DESCRIPTION
**Pull request by borismelnik**
Fixes: #9655

### What does the PR do

Rename `Edit` type to `ContactRequest` and use it type as Text message

### Affected areas

Chat

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [ ] I've checked the design and this PR matches it

Снимок экрана 2023-03-07 в 18 15 51

### CODE REVIEW

**Review by caybro — APPROVED** [review]
LGTM

**Review by 0x-r4bbit — APPROVED** [review]

**Review by mprakhov — APPROVED** [review]

File: src/app_service/common/types.nim [hunk 1]
```diff
@@ -14,7 +14,7 @@ type
     Audio = 8
     Community = 9
     Gap = 10
-    Edit = 11
+    ContactRequest = 11
```
Comment (jrainville) [inline src/app_service/common/types.nim:17]: So, what's the int for Edit then? It should still be supported

**Review by jrainville — COMMENTED** [review]
If possible, add the int for Edit as well, even if it's not used, it's good to know the options

File: src/app_service/common/types.nim (hunk 1 above)
Comment (borismelnik) [reply src/app_service/common/types.nim:17]: I don't see any types named as `Edit` in status-go. In our code it was not used.

**Review by jrainville — APPROVED** [review]

**Pull request merged**